campaign-launcher/interfaceV2/src/utils/index.ts

```diff
@@ -155,3 +155,17 @@ export const calculateHash = async (manifest: string, algorithm: AlgorithmIdenti
   const hashArray = Array.from(new Uint8Array(hashBuffer));
   return hashArray.map(b => b.toString(16).padStart(2, '0')).join('');
 };
+
+export const filterFalsyQueryParams = (
+  params: Record<string, string | number | undefined>
+): Record<string, string | number> => {
+  const result: Record<string, string | number> = {};
+  
+  for (const [key, value] of Object.entries(params)) {
+    if (value) {
```

Comment:
What if my value is `0` int or `false` boolean?

Reply:
@dnechay the value would be filtered out. can't come up with a case, when a frontend should pass `0` or `false` in

Reply:
@KirillKirill at the moment we don't have such query params, but in general it can be, so better to have check like
```
if ([undefined, null, ''].includes(value))
```
because these values might come from forms or components